File: loianeTraining/javaBasico/exercicios-aulas-18-19/Ex9.java
public class Ex9 {
    public static void main(String[] args) {
        int[] a = { 11, 23, 35, 47, 59, 611, 713, 815, 197, 149 };
        int[] b = { 1, 2, 4, 6, 8, 10, 12, 14, 16, 18 };
        double[] c = new double[10];

        for (int i = 0; i < a.length; i++) {
            c[i] = a[i] / b[i];
            System.out.println(c[i] + " = " + a[i] + " / " + b[i]);
        }
    }
}

File: Loiane_Training_C#/Basico/C/Ex9.cs
using System;

public class Ex9
{
    static void Main(string[] args)
    {

        Console.WriteLine("Digite 3 numeros");

        double numero1 = double.Parse(Console.ReadLine());
        double numero2 = double.Parse(Console.ReadLine());
        double numero3 = double.Parse(Console.ReadLine());

        double maior = 0;
        double menor = 0;
        double mediana = 0;

        if (numero1 > numero2 && numero1 > numero3)
        {
            maior = numero1;
        }
        else if (numero2 > numero1 && numero2 > numero3)
        {
            maior = numero2;
        }
        else
        {
            maior = numero3;
        }

        if (numero1 < numero2 && numero1 < numero3)
        {
            menor = numero1;
        }
        else if (numero2 < numero1 && numero2 < numero3)
        {
            menor = numero2;
        }
        else
        {
            menor = numero3;
        }

        if (numero1 < maior && numero1 > menor)
        {
            mediana = numero1;
        }
        else if (numero2 < maior && numero2 > menor)
        {
            mediana = numero2;
        }
        else
        {
            mediana = numero3;
        }
        Console.WriteLine(maior + ", " + mediana + ", " + menor);
    }
}
